spec/controllers/administrateurs/groupe_instructeurs_controller_spec.rb

```diff
@@ -995,4 +995,41 @@ def remove_instructeur(instructeur)
       expect(gi_1_1.signature).to be_attached
     end
   end
+
+  describe '#update_hide_instructeurs_email' do
+    let(:administrateur) { create(:administrateur) }
```

Comment:
micro optimisation, on peut utiliser les fixtures pour cet admin
```suggestion
    let(:administrateur) { administrateurs(:default_admin) }
```